wp-content/themes/pub/wporg-learn-2020/css/components/_section-heading.scss

```
@@ -11,6 +11,11 @@
 			display: block;
 			font-size: 1rem;
 		}
+
+		&.bigger {
```

From a design consistency standpoint I think this should be applied sitewide via the existing class applied to these headings:
```suggestion
		&.h2 {
```